shell/platform/android/android_egl_surface.cc

```diff
@@ -60,6 +60,17 @@ void LogLastEGLError() {
   FML_LOG(ERROR) << "Unknown EGL Error";
 }
 
+namespace {
+
+static bool HasExtension(const char* extensions, const char* name) {
```

Comment:
We have this method in like 3 or 4 places now.  Or maybe we will with the swiftshader patch. We should consider moving it somewhere common

Reply:
SGTM